apps/web/src/pages/Applications/ApplicantDashboardPage/components/TrackApplications/TrackApplications.tsx

```diff
@@ -0,0 +1,286 @@
+                  />
+                ))
+              ) : (
+                <Well data-h2-text-align="base(center)">
+                  <p
+                    data-h2-font-size="base(h6)"
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin="base(0, 0, x.25, 0)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Applications that are in progress will appear here.",
+                      id: "C9PbZN",
+                      description:
+                        "Text displayed in the applications in progress section when empty.",
+                    })}
+                  </p>
+                  <a href={paths.browsePools()}>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Check out available opportunities to start an application.",
+                      id: "oSPdz1",
+                      description:
+                        "Additional text displayed in the applications in progress section when empty.",
+                    })}
+                  </a>
+                </Well>
+              )}
+            </AnimatedContent>
+          </Accordion.Item>
+          {/* past applications */}
+          <Accordion.Item value="past">
+            <StandardAccordionHeader headingAs="h3">
+              {currentAccordionItems.includes("past")
+                ? intl.formatMessage(
+                    {
+                      defaultMessage:
+                        "Hide past applications ({applicationCount})",
+                      id: "Vq97zy",
+                      description:
+                        "Heading for past applications accordion on the applicant dashboard.",
+                    },
+                    {
+                      applicationCount: pastApplications.length ?? "0",
+                    },
+                  )
+                : intl.formatMessage(
+                    {
+                      defaultMessage:
+                        "Show past applications ({applicationCount})",
+                      id: "mYD993",
+                      description:
+                        "Heading for past applications accordion on the applicant dashboard.",
+                    },
+                    {
+                      applicationCount: pastApplications.length ?? "0",
+                    },
+                  )}
+            </StandardAccordionHeader>
+            <AnimatedContent isOpen={currentAccordionItems.includes("past")}>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This section contains old applications that have been fully assessed, as well as applications that have missed the submission deadline.",
+                id: "mH2eSA",
+                description:
+                  "Introductory text displayed in past applications section.",
+              })}
```

Comment:
Same thing with the other `subtitle`

Reply:
:+1: [remove redundant attribute](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6823/commits/1f7a8d2c6ef34bdf20e4477ba4b6357b8fdcf43a)